Packs/Accessdata/Integrations/AccessdataV2/AccessdataV2.py

```diff
@@ -0,0 +1,361 @@
+        raise ValueError(f"Failed to gather case with id ({caseid}).")
+
+    agent = Agent(case, target)
+    job = agent.acquire_disk(**kwargs)
+
+    return CommandResults(
+        outputs_prefix="Accessdata.Case.Job",
+        outputs={
+            "ID": job["id"],
+        },
+        outputs_key_field="ID",
+        readable_output=tableToMarkdown("Job", dict(job))
+    )
+
+
+_comparator_mapping = {
+    "==": "__eq__",
+    "!=": "__ne__",
+    ">": "__gt__",
+    "<": "__lt__",
+    ">=": "__ge__",
+    "<=": "__le__"
+}
+
+
+@wrap_demisto_command("accessdata-api-create-filter")
+def _create_filter(client, column, comparator, value):
+    comp = _comparator_mapping.get(comparator, comparator)
+    # get the relevant attribute
+    attr = client.attributes.first_matching_attribute("attributeUniqueName", column)
+    # if not exists, raise error
+    if not attr:
+        raise ValueError(f"Cannot find attribute of name ({column}).")
+    # try get comparison method
+    has_comp = hasattr(attr, comp)
+    if not has_comp:
+        raise ValueError(f"Cannot find compare method of method name ({comp}).")
+    # compare and return
+    filter_json = getattr(attr, comp)(value)
+    return CommandResults(
+        outputs_prefix="Accessdata",
+        outputs={
+            "Filter": dumps(filter_json)
+        },
+        outputs_key_field="Filter",
+        readable_output=tableToMarkdown("Filter", filter_json)
+    )
+
+
+@wrap_demisto_command("accessdata-api-combine-filter-and")
+def _and_filter(client, filter_json1, filter_json2):
+    # try find json in the filters
+    try:
+        filter_json1 = loads(filter_json1)
+        filter_json2 = loads(filter_json2)
+    except Exception:
+        raise ValueError("Both filters must be JSON content.")
+
+    filter_json = and_(filter_json1, filter_json2)
+    return CommandResults(
+        outputs_prefix="Accessdata",
+        outputs={
+            "Filter": dumps(filter_json)
+        },
+        outputs_key_field="Filter",
+        readable_output=tableToMarkdown("Filter", filter_json)
+    )
+
+
+@wrap_demisto_command("accessdata-api-combine-filter-or")
+def _or_filter(client, filter_json1, filter_json2):
+    # try find json in the filters
+    try:
+        filter_json1 = loads(filter_json1)
+        filter_json2 = loads(filter_json2)
+    except Exception:
+        raise ValueError("Both filters must be JSON content.")
+
+    filter_json = or_(filter_json1, filter_json2)
+    return CommandResults(
+        outputs_prefix="Accessdata",
+        outputs={
+            "Filter": dumps(filter_json)
+        },
+        outputs_key_field="Filter",
+        readable_output=tableToMarkdown("Filter", filter_json)
+    )
+
+
+@wrap_demisto_command("test-module")
+def _test_module(client):
+    # test the client can reach the case list
+    try:
+        client.cases
+    except Exception:
+        raise ValueError("Client cannot reach the server with the current configuration.")
```

Comment:
consider raising a DemistoException and pass the original exception as an argument, to give users/debuggers a hint of what went wrong

Reply:
Agreed that this makes more sense and will update respectively 👍

Reply:
Please find addressed here: https://github.com/demisto/content/pull/16390/commits/656b3cb1a31fa4426b53b31e2105112e02fbe2fa